ONSdigital/ras-deploy
# Add respondent home ui

# Motivation and Context

Adds the respondent-home-ui app to the pipeline.

# What has changed

* `pipeline.yml` and `loadtest.yml` - New app config.
* `tasks/respondent-home-ui/bundle-respondent-home-ui.yml` - Bundle the app before being deployed. This is to allow the ONS internally-hosted Ruby Gems to be included when the app is pushed by Cloud Foundry
* `secrets.yml.example` - Updated this template with new keys. N.B. Main secrets file will also be updated to reflect.
* When this PR is merged with master, the pipeline can be updated with this changed config.



# How to test?

* Check the `pipeline.yml` file for any errors or anything which looks out of place.



# Links

[Trello](https://trello.com/c/SwkUMf9O/99-dev-task-7-ext-access-eq-social-survey-rh-8)
